### PR TITLE
Adding Texas_VSON-HR_1.5x2mm_P0.5mm

### PR DESCRIPTION
**Pull request by ki5libs**
Adding Texas_VSON-HR_1.5x2mm_P0.5mm
[Datasheet](http://www.ti.com/lit/ds/symlink/tps62823.pdf)

### CODE REVIEW

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml [hunk 1]
```diff
@@ -587,6 +587,29 @@ Broadcom_DFN-6_2.0x2.0mm:
...
   num_pins_y: 3
 
+Texas_VSON-HR_1.5x2mm_P0.5mm:
+  device_type: 'VSON'
+  size_source: 'http://www.ti.com/lit/ds/symlink/tps62823.pdf'
+  manufacturer: 'Texas'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  body_size_x:
+    nominal: 1.5
+    tolerance: 0.1
```
Comment (poeschlr) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml]: The datasheet uses min/max dimensioning so use the same style:
```yaml
body_size_x:
  minimum: 1.4
  maximum: 1.6
```

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml [hunk 2]
```diff
@@ -587,6 +587,29 @@ Broadcom_DFN-6_2.0x2.0mm:
   num_pins_x: 0
   num_pins_y: 3
 
+Texas_VSON-HR_1.5x2mm_P0.5mm:
+  device_type: 'VSON'
+  size_source: 'http://www.ti.com/lit/ds/symlink/tps62823.pdf'
```
Comment (poeschlr) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml]: include the page in the link:
http://www.ti.com/lit/ds/symlink/tps62823.pdf#page=29

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml (hunk 1 above)
Comment (ki5libs) [reply scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml]: Done. And also for size_y

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml (hunk 2 above)
Comment (ki5libs) [reply scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml]: Done

**Comment by poeschlr** [comment]
thanks

**Pull request merged**